QueryCiv3/BiqSections/Tile.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace QueryCiv3.Biq
+{
+    [StructLayout(LayoutKind.Sequential, Pack=1)]
+    public unsafe struct TILE
+    {
+        // Not 100% sure the flags are correct. The Apolyton documentation gets confusing here
```

Comment:
Thanks for calling this out.  It's always better to note where there's uncertainty, and in this case it made sure I compared to both the Apolyton docs and the documentation I had accumulated over the years in editor code.